File: SystemsProgramming/guess.c
#include <stdlib.h>
#include <stdio.h>
#include <time.h>
    
int main(){
  int number, guess;
  srand( time( 0 ) );
  number = rand() % (10 + 1);

  printf( "Guess what the number is: " );
    
  while( scanf( "%d", &guess ) == 1 ) {
    if( number == guess ){
      printf( "You got it!\n" );
      break;
    }
    printf( "Your guess was too %s.\nGuess again: ",
      number < guess ? "high" : "low" );
  }
}

File: SystemsProgramming/Guess.java
import java.util.Random; import java.util.Scanner;
public class Guess {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    Random random = new Random();
    int randomNumber = random.nextInt(10 + 1);
    int guessedNumber = 0;
 
    do {
      System.out.print("Guess what the number is: ");
      guessedNumber = scan.nextInt();
      if (guessedNumber > randomNumber)
        System.out.println("Your guess is too high!");
      else if (guessedNumber < randomNumber)
        System.out.println("Your guess is too low!");
      else
        System.out.println("You got it!");
    } while (guessedNumber != randomNumber);
  }
}